en/docs/apis/keystore-rest-api.md

```diff
@@ -0,0 +1,43 @@
+---
+template: templates/swagger.html
+---
+
+# Keystore Management API Definition - v1
+
+!!! Note 
+    Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server (WSO2 IS). 
+    To try some APIs, a tenant needs to be created with the domain name, 'wso2.com'. See [Adding a new tenant](../administer/adding-new-tenant.md) for more information.
```

Comment:
Please make the link in the standard format with a TO-DO instead of leaving it linked to the previous version so that it's easier when we need to link it.